code/obj/artifacts/artifact_objects/bombs.dm

```diff
@@ -290,3 +291,177 @@ ABSTRACT_TYPE(/datum/artifact/bomb)
+
+/datum/artifact/bomb/transmute
+	associated_object = /obj/machinery/artifact/bomb/transmute
+	validtypes = list("wizard","eldritch","martian","ancient","precursor")
+	rarity_class = 4
+	react_xray = list(17,95,95,3,"ANOMALOUS")
+	warning_initial = ""
+	warning_final = ""
+	alarm_initial = null
+	alarm_during = null
+	alarm_final = "sound/machines/satcrash.ogg"
+	var/material = "gold"
+	var/datum/material/mat = null
+	var/affects_organic = 0 // 1 means material human, 2 means material statue
+	var/range = 7
+	var/smoothEdge = 0 // with this, the area will be perfectly circular
+
+	post_setup()
+		affects_organic = pick(
+			prob(5); 0, // does nothing
+			prob(2); 1, // material human
+			prob(2); 2) // material statue
+		smoothEdge = pick(0,1)
+
+		explode_delay = rand(30 SECONDS, 2 MINUTES)
+		switch(artitype.name)
+			if("wizard") // magical stuff
+				material = pick(
+					100;"gold",
+					100;"syreline",
+					100;"silver",
+					100;"cobryl",
+					50;"miracle",
+					50;"soulsteel",
+					50;"hauntium",
+					50;"ectoplasm",
+					10;"ectofibre",
+					10;"wiz_quartz",
+					10;"wiz_topaz",
+					10;"wiz_ruby",
+					10;"wiz_amethyst",
+					10;"wiz_emerald",
+					10;"wiz_sapphire",
+					10;"starstone")
+			if("eldritch") // fuck you
+				material = pick(
+					100;"koshmarite",
+					100;"plasmastone",
+					100;"telecrystal",
+					50;"erebite")
+			if("martian") // organic stuff
+				material = pick(
+					100;"flesh",
+					100;"viscerite",
+					100;"leather",
+					100;"cotton",
+					100;"coral",
+					50;"spidersilk",
+					50;"beewool",
+					50;"beeswax",
+					50;"chitin",
+					50;"bamboo",
+					50;"wood",
+					50;"bone",
+					20;"blob",
+					20;"pizza",
+					2;"butt")
+			if("ancient") // industrial type stuff
+				material = pick(
+					100;"electrum",
+					100;"steel",
+					100;"mauxite",
+					100;"copper",
+					100;"pharosium",
+					100;"glass",
+					100;"char",
+					100;"molitz",
+					50;"molitz_b",
+					50;"bohrum",
+					50;"cerenkite",
+					50;"plasmasteel",
+					50;"claretine",
+					50;"plasmaglass",
+					50;"uqill",
+					50;"latex",
+					50;"synthrubber",
+					50;"synthblubber",
+					50;"synthleather",
+					50;"fibrilith",
+					30;"carbonfibre",
+					30;"diamond",
+					30;"dyneema",
+					10;"iridiumalloy",
+					1;"neutronium")
+			if("precursor") // uh, the rest
+				material = pick(
+					100;"rock",
+					100;"slag",
+					100;"ice",  // no possible way this ends poorly
+					15;"spacelag",
+					15;"cardboard",
+					15;"frozenfart",
+					15;"negativematter")
+
+		mat = getMaterial(material)
+
+		warning_initial = "appears to be turning into [mat.name]."
+		warning_final = "begins transmuting nearby matter into [mat.name]!"
+
+		var/matR = GetRedPart(mat.color)
+		var/matG = GetGreenPart(mat.color)
+		var/matB = GetBluePart(mat.color)
+		lightColor = list(matR, matG, matB, 255)
+
+		// recharge_delay = rand(1 MINUTE, 5 MINUTES)
```

Comment:
What's up with this commented out line?

Reply:
I had plans to maybe make them reuseable, which would have used the recharge_delay thing.
But then I realized that my other PR that made that var a bomb thing instead of just a chembomb thing wasn't merged yet, so I shelved it.